AssemblyInfo.Shared.cs

```diff
@@ -26,8 +26,8 @@
 [assembly: AssemblyFileVersion("9.1.0.0")]
 [assembly: AssemblyInformationalVersion("Version:9.1.0.0 Branch:not-set Sha1:not-set")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("SonarSource and Microsoft")]
-[assembly: AssemblyCopyright("Copyright © SonarSource and Microsoft 2015-2023")]
+[assembly: AssemblyCompany("SonarSource")]
```

Comment:
There's also a mention in the all 3 nuspec files nested in this directory
https://github.com/SonarSource/sonar-scanner-msbuild/tree/master/nuspec